### PR TITLE
Rm chimerax

### PR DESCRIPTION
**Pull request by rmarabini**
Adapt spider plugin to chimerax.  
Requires PR scipion-em:scipio-em  Rm chimerax #132  (https://github.com/scipion-em/scipion-em/pull/132)

### CODE REVIEW

File: spider/viewers/viewer_refinement.py [hunk 1]
```diff
@@ -186,21 +186,17 @@ def _showVolumesChimera(self):
...
-                # We assume that the chimera script will be generated
-                # at the same folder than spider volumes
-                localVol = os.path.basename(vol)
-                if os.path.exists(vol):
-                    f.write("open %s format spider\n" % localVol)
-            f.write('tile\n')
-            f.close()
-            view = ChimeraView(cmdFile)
-        else:
-            view = ChimeraClientView(volumes[0])
```
Comment (azazellochg) [inline spider/viewers/viewer_refinement.py:202]: Thanks, @rmarabini . Is ChimeraClientView then gone completely?

File: spider/viewers/viewer_refinement.py [hunk 2]
```diff
@@ -30,7 +30,7 @@
 import pyworkflow.protocol.params as params
 from pyworkflow.viewer import DESKTOP_TKINTER, WEB_DJANGO
 from pwem.viewers import (EmPlotter, ChimeraClientView, ChimeraView,
-                          EmProtocolViewer, ChimeraAngDist)
+                          EmProtocolViewer, ChimeraAngDist, pwutils)
```
Comment (azazellochg) [inline spider/viewers/viewer_refinement.py]: that's wrong :) Actually, you don't need pwutils in this file

File: spider/viewers/viewer_refinement.py (hunk 1 above)
Comment (rmarabini) [reply spider/viewers/viewer_refinement.py:202]: Yes, @azazellochg It is gone

File: spider/viewers/viewer_refinement.py (hunk 2 above)
Comment (rmarabini) [reply spider/viewers/viewer_refinement.py]: You are right @azazellochg I do not know how pwutils ended in that line.  I remove pwutils form the import line.

**Pull request merged**